webapp/src/components/Footer/index.js

```diff
@@ -1,10 +1,7 @@
 import React, { memo } from 'react'
-import Grid from '@mui/material/Grid'
-import List from '@mui/material/List'
-import ListItemText from '@mui/material/ListItemText'
-import ListItem from '@mui/material/ListItem'
-import Box from '@mui/material/Box'
+import { Grid, Box } from '@mui/material'
```

Comment:
according MUI docs this kind of import components `import Box from '@mui/material/Box'` is better than `import { Grid, Box } from '@mui/material'`
Please import de component directly.

Actually we are avoiding use Box and Grid components in order to keep up the performance